backbone.js

```diff
@@ -1994,9 +1994,31 @@
       if (this.fragment === decodedFragment) return;
       this.fragment = decodedFragment;
 
+      featuredItemOverride: {
```

Comment:
lets make sure the tests are updated to cover this new code. you should be able to `npm install` and `npm test`